package.json

```diff
@@ -15,19 +15,19 @@
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
     "@bcrs-shared-components/action-chip": "1.0.6",
-    "@bcrs-shared-components/bread-crumb": "1.0.1",
+    "@bcrs-shared-components/breadcrumb": "2.0.1",
     "@bcrs-shared-components/certify": "1.0.18",
-    "@bcrs-shared-components/completing-party": "1.0.1",
+    "@bcrs-shared-components/completing-party": "2.0.0",
```

Comment:
Major version update due to large refactor while finishing its implementation. This way, if anyone is using v1, they will know to expect changes with v2.